ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -16,14 +16,27 @@ then
   exit 0
 fi
 
+# Temporarily limit the number of jobs we're going to ingest into loki while we test and scale the new instance:
+if [[ $JOB_NAME != "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-single-node" ]] \
+	&& [[ $JOB_NAME != "periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-8-ovn" ]] \
+	&& [[ ! "$JOB_NAME" =~ .*gcp.* ]] \
+	&& [[ $JOB_NAME != "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-aws-ovn-arm64-single-node" ]] \
+	&& [[ $JOB_NAME != "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-remote-libvirt-s390x" ]] \
+	&& [[ $JOB_NAME != "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-remote-libvirt-s390x" ]]; then
+	echo "This job is not on the list of supported jobs we're testing for the new loki, skipping..."
+	exit 0
```

Comment:
if we're going to incrementally add jobs, we can do something like this for readability, etc.:

```bash
SUPPORTED_JOBS=(
  "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-single-node"
  "periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-8-ovn"
  ".*gcp*."
  "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-aws-ovn-arm64-single-node"
  "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-remote-libvirt-s390x"
  "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-remote-libvirt-s390x"
)

is_supported_job() {
  job_name=$1
  for supported_job in ${SUPPORTED_JOBS[@]}; do
    if [[ "$job_name" =~ $supported_job ]]; then
      echo "1"
      return
    fi
  done
  echo "0"
}

if ! is_supported_job $JOB_NAME; then
  echo "This job is not on the list of supported jobs we're testing for the new loki, skipping..."
  exit 0
fi
```

You can try these unit tests to ensure this is working:

```bash
t=$(is_supported_job "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-single-node")
echo $t
t=$(is_supported_job "periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-8-ovn")
echo $t
t=$(is_supported_job ".*gcp*.")
echo $t
t=$(is_supported_job "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-aws-ovn-arm64-single-node")
echo $t
t=$(is_supported_job "periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-remote-libvirt-s390x")
echo $t
t=$(is_supported_job "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-remote-libvirt-s390x")
echo $t
t=$(is_supported_job "my-famous-gcp-job")
echo $t
t=$(is_supported_job "my-famous-aws-job")
echo $t
```